ctapipe/io/hdf5tableio.py

```diff
@@ -148,6 +148,9 @@ class Schema(tables.IsDescription):
         # create pytables schema description for the given container
         pos = 0
         for container in containers:
+
+            container.validate()  # ensure the data are complete
```

Comment:
`validate` checks that every `Field` in the container is of the expected type?

Reply:
See #1301